<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="84081" started-at="2019-12-31T13:25:40 IST" finished-at="2019-12-31T13:27:04 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="84081" started-at="2019-12-31T13:25:40 IST" finished-at="2019-12-31T13:27:04 IST">
      <class name="testcases.LoginTestActiTime">
        <test-method status="PASS" signature="setUpSuite()[pri:0, instance:testcases.LoginTestActiTime@3224f60b]" name="setUpSuite" is-config="true" duration-ms="2938" started-at="2019-12-31T13:25:37 IST" finished-at="2019-12-31T13:25:40 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpSuite -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:testcases.LoginTestActiTime@3224f60b]" name="setUp" is-config="true" duration-ms="36119" started-at="2019-12-31T13:25:40 IST" finished-at="2019-12-31T13:26:16 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="pageLoad()[pri:0, instance:testcases.LoginTestActiTime@3224f60b]" name="pageLoad" is-config="true" duration-ms="466" started-at="2019-12-31T13:26:17 IST" finished-at="2019-12-31T13:26:17 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pageLoad -->
        <test-method status="FAIL" signature="loginApp1()[pri:1, instance:testcases.LoginTestActiTime@3224f60b]" name="loginApp1" duration-ms="12851" started-at="2019-12-31T13:26:17 IST" finished-at="2019-12-31T13:26:30 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testcases.LoginTestActiTime.loginApp1(LoginTestActiTime.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp1 -->
        <test-method status="PASS" signature="takePhoto(org.testng.ITestResult)[pri:0, instance:testcases.LoginTestActiTime@3224f60b]" name="takePhoto" is-config="true" duration-ms="7066" started-at="2019-12-31T13:26:30 IST" finished-at="2019-12-31T13:26:37 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginApp1 status=FAILURE method=LoginTestActiTime.loginApp1()[pri:1, instance:testcases.LoginTestActiTime@3224f60b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takePhoto -->
        <test-method status="PASS" signature="pageLoad()[pri:0, instance:testcases.LoginTestActiTime@3224f60b]" name="pageLoad" is-config="true" duration-ms="2835" started-at="2019-12-31T13:26:37 IST" finished-at="2019-12-31T13:26:40 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pageLoad -->
        <test-method status="PASS" signature="loginApp()[pri:2, instance:testcases.LoginTestActiTime@3224f60b]" name="loginApp" duration-ms="21595" started-at="2019-12-31T13:26:40 IST" finished-at="2019-12-31T13:27:02 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp -->
        <test-method status="PASS" signature="takePhoto(org.testng.ITestResult)[pri:0, instance:testcases.LoginTestActiTime@3224f60b]" name="takePhoto" is-config="true" duration-ms="1538" started-at="2019-12-31T13:27:02 IST" finished-at="2019-12-31T13:27:03 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginApp status=SUCCESS method=LoginTestActiTime.loginApp()[pri:2, instance:testcases.LoginTestActiTime@3224f60b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takePhoto -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.LoginTestActiTime@3224f60b]" name="tearDown" is-config="true" duration-ms="1262" started-at="2019-12-31T13:27:03 IST" finished-at="2019-12-31T13:27:04 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.LoginTestActiTime -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
